Frontend-Angular/tsconfig.json

```diff
@@ -0,0 +1,33 @@
+/* To learn more about this file see: https://angular.io/config/tsconfig. */
+{
+  "compileOnSave": false,
+  "compilerOptions": {
+    "baseUrl": "./",
+    "outDir": "./dist/out-tsc",
+    "forceConsistentCasingInFileNames": true,
+    "strict": true,
```

Comment:
we can mesh this up, let's see if candidate will make it better? i.e. remove this ` "strict": true,`